roles/pulp_common/tasks/configure.yml

```diff
@@ -7,6 +7,7 @@
         owner: root
         group: '{{ pulp_group }}'
         mode: 0750
+      notify: Restore SELinux contexts on Pulp dirs that must exist
```

Comment:
Wow, that's a hell of a handler name...

Reply:
"always already exist" might be more descriptive.